tests/constants/githubcomment.constants.ts

```diff
@@ -15,6 +15,64 @@ export const EMPTY_TEAM_RESPONSE: GithubComment = {
   }
 };
 
+export const TEAM_RESPONSE_SEVERITY_DISAGREEMENT: GithubComment = {
+  body:
+    "# Team's Response\r\n" +
```

Comment:
Can use backtick (`` `  ``) instead of quotes, to create the multiline string neatly
(i.e. don't have to concatenate each line)

Reply:
Got it

Reply:
Hi @anubh-v , when I changed it to use multi line string with back ticks as shown below, the parser no longer works.
![image](https://user-images.githubusercontent.com/59414467/127778806-92a55726-4b2d-4c40-9e0b-013f9329110b.png)

My guess is because I've to add the `\` escape character for the backticks in of `severity.low` and `type.featureFlaw`, causing the regex to fail.

Regex:
![image](https://user-images.githubusercontent.com/59414467/127778800-cbc8c8b4-4ae9-41f4-a630-b8025e325b1f.png)

As such, I'm not sure if we can use backticks for the test data without changing the regex matcher and altering the parsing of the real comment data. Do you have any ideas on how we can resolve this problem or should we go with another approach?

Note:
I suspect that the other Github comment test data which was previously implemented by someone, used concatenation of strings for this issue too

Reply:
I see. Ok, we can keep the existing style. But we do not need both `\r` and `\n` in the comment body, let's just keep `\n` ?

Reply:
Sure, will remove the \r